### PR TITLE
tests pass for sortArrowIcon and sortArrowAnimationDuration

### PR DESCRIPTION
**Pull request by hootyjeremy**
The tests are passing in this one for me. This one should be cleaner. The only thing I'm not sure about is why `activeColor` and `checkColor` are showing as additions in `lib/src/data_table_2.dart` since I didn't make that change. 

image

### CODE REVIEW

**Comment by maxim-saplin** [comment]
> `activeColor` and `checkColor`

These ones has recently been removed from main... I think it is a poor idea to create PR for every trouble, now we can see a side effect. Rather than merge main into previous commit you could have bugs creeping into these commit by returning code that was deliberately removed. I believe you simply rewritten the whole file in data_table_2.dart with your old version from days back effectively removing whatever changes there were in main branch

**Comment by hootyjeremy** [comment]
I'll admit I don't understand versioning with other people's commits and all this. After I saw that I had somehow wrecked the whole thing this morning, I thought I should just re-fork the current version and add my own updates but that is apparently not what happened. My workflow with GitHub is probably not optimal but this is all new to me. I don't know how scrapping my failed efforts from yesterday and starting with a fresh fork and adding my own changes did this.

I just want to choose my own icon and have an instant toggle. I don't fully grasp GitHub and versions between multiple parties and this is taking away a lot of time from my own actual project. I'm glad for the learning experience but I've only just added a few lines and it has taken a week and I've had to learn more about testing which is fine but I have no idea how to contribute this cleanly. Should I make another attempt?

File: lib/src/data_table_2.dart [hunk 1]
```diff
@@ -267,6 +272,8 @@ class DataTable2 extends DataTable {
         ),
         child: Center(
           child: Checkbox(
+            activeColor: themeData.colorScheme.primary,
```
Comment (hootyjeremy) [inline lib/src/data_table_2.dart]: I removed these two in the next commit since they were accidentally added.

**Comment by hootyjeremy** [comment]
Sorry about the confusion yesterday. I have removed activeColor and checkColor after I somehow mistakenly re-introduced the with this PR. At this point, I believe the PR might be clean.

**Comment by maxim-saplin** [comment]
There's too much noise in this PR, some unknown to me changes on my closing. Closing it as I don't have time for the investigations and fixing the discrepancies

**Comment by maxim-saplin** [comment]
If you still want to complete the PR, than create a new one from the main branch and manually copy there the parts of files you changed rather than completely rewriting filled with whatever local copies you might have. Besides I expect that you'll have all the tests in place, up and running. The code being compilable is smith that I doesn't need to be discussed

**Pull request closed without merging**